db/model/model.js

```diff
@@ -46,35 +46,36 @@ exports.fetchArticleByID = (article_id) => {
 };
 
 exports.fetchAllArticles = () => {
-  return (
-    db
-      .query(
-        `SELECT articles.author, articles.title,articles.topic, articles.article_id,articles.topic,
+  return db
+    .query(
+      `SELECT articles.author, articles.title,articles.topic, articles.article_id,articles.topic,
         articles.created_at, articles.votes, articles.article_img_url,
         comments.article_id AS comment_count
         FROM articles 
         INNER JOIN  
         comments ON comments.article_id = articles.article_id
-        ORDER BY comments.created_at DESC;`
-      )
-      // .then((articles) => {
-      //   const removeBody = articles.map((article) => {
-      //     delete article.body;
-      //     return articles;
-      //   });
-      //   return removeBody;
-      // })
-      .then(({ rows }) => {
-        // console.log(rows);
-        return rows;
-      })
-      .catch((err) => {
-        throw err;
-      })
-  );
+        ORDER BY articles.created_at DESC;`
+    )
+    .then(({ rows }) => {
+      return rows;
+    })
+    .catch((err) => {
+      throw err;
+    });
+};
+
+exports.fetchCommentsByArticleId = (article_id) => {
+  return db
+    .query(
+      `
+  SELECT * FROM comments WHERE article_id = $1 
+  ORDER BY comments.created_at DESC;`,
+      [article_id]
+    )
+    .then(({ rows }) => {
+      return rows;
+    })
+    .catch((err) => {
```

Comment:
This catch block isn't needed because there's already one on the controller, earlier in the promise chain